src/plugins/data/public/public.api.md

```diff
@@ -39,12 +40,15 @@ import { KibanaClient } from '@elastic/elasticsearch/api/kibana';
 import { KibanaConfigType } from 'src/core/server/kibana_config';
 import { Location } from 'history';
 import { LocationDescriptorObject } from 'history';
+import { Logger } from '@kbn/logging';
+import { LogMeta } from '@kbn/logging';
```

Comment:
@elastic/kibana-app-arch The additional imports in `src/plugins/data/public/public.api.md` and `src/plugins/data/server/server.api.md` should be the only changes impacting you